test/distributed_exec.jl

```diff
@@ -227,29 +208,42 @@ put!(f, :OK)
 @test remotecall_fetch(k->haskey(Distributed.PGRP.refs, k), wid1, fid) == false
 @test fetch(f) === :OK
 
-# RemoteException should be thrown on a put! when another process has set the value
-f = Future(wid1)
-fid = remoteref_id(f)
+max_attempts = 1000
```

Comment:
Can the extra debugging code be removed? I think at least `max_attempts` shouldn't be 1000.

Reply:
I've added an explanation and reduced it to 100